compiler/rustc_ast_pretty/src/pprust/state.rs

```diff
@@ -1336,6 +1336,9 @@ impl<'a> State<'a> {
                 self.print_outer_attributes(&loc.attrs);
                 self.space_if_not_bol();
                 self.ibox(INDENT_UNIT);
+                if loc.super_.is_some() {
+                    self.word_nbsp("let");
```

Comment:
```suggestion
                    self.word_nbsp("super");
```
:thinking: am i missing something here :sweat_smile: